src/bundle/Resources/views/fieldtypes/edit/ezselection.html.twig

```diff
@@ -11,6 +11,9 @@
         {% endfor %}
     </ul>
     <ul class="ez-data-source__options ez-data-source__options--hidden">
+        {%- if placeholder is not none -%}
```

Comment:
Here, you do not seem to manage if the field is required or not.

Will it be possible not to set any value ?

Reply:
It will work for required field - it will select first value by default as required field cannot be empty.

It won't be possible to not set any value as the validation will kick in.